hyperspy/drawing/utils.py

```diff
@@ -724,10 +749,6 @@ def plot_images(images,
     ax_im_list = [0] * len(isrgb)
     # Loop through each image, adding subplot for each one
     for i, ims in enumerate(images):
-        # Get handles for the signal axes and axes_manager
-        axes_manager = ims.axes_manager
```

Comment:
Thanks for the explanation @to266. When I have got a bit of time I will come back to this.